scripts/output/single/reporting.R

```diff
@@ -84,18 +84,24 @@ if(file.exists(edgetOutputDir)) {
   }
   message("start generation of EDGE-T reporting")
   EDGET_output <- reportEdgeTransport(edgetOutputDir,
-                                      isTransportExtendedReported = FALSE, 
+                                      isTransportExtendedReported = FALSE,
                                       modelName = "REMIND",
-				      scenarioName = scenario,
+				                              scenarioName = scenario,
                                       gdxPath = file.path(outputdir, "fulldata.gdx"),
                                       isStored = FALSE)
 
+  REMINDoutput <- as.data.table(read.quitte(file.path(outputdir, paste0("REMIND_generic_", scenario,"_withoutPlus.mif"))))
+  sharedVariables <- EDGET_output[variable %in% REMINDoutput$variable | grepl(".*edge", variable)]
+  EDGET_output <- EDGET_output[!variable %in% REMINDoutput$variable]
+  message("The following variables will be dropped from the EDGE-Transport reporting because
+                they are in the REMIND reporting: ", paste(unique(sharedVariables$variable), collapse = ", "))
```

Comment:
I struggle a bit understanding your code here. Why do you add `grepl(".*edge", variable)` to the `sharedVariables`, as they are obviously not shared but EDGE-T only. Then, you drop all available in REMIND, but claim in the message to have dropped the `edge` ones as well, which you didn't. That confuses me.

Reply:
Yes, it's more of a quick fix than a long-term solution. There are some variables where we have added an “edge” to the variable name because the variable is also reported by remind2.  So we want to filter out those that are duplicates by name and those that have an “edge” in the name. I decided to filter out the "edge" variables as well, so that no one can mistakenly use them for project reportings as we now have the inconsistency with the bunkers reporting between remind2 and edgeTransport

Reply:
But your code would only actually filter them out if you did `EDGET_output <- EDGET_output[!variable %in% sharedVariables]`